src/map/script.c

```diff
@@ -17243,15 +17269,18 @@ BUILDIN(unitstop) {
 
 /// Makes the unit say the message
 ///
-/// unittalk <unit_id>,"<message>";
+/// unittalk <unit_id>,"<message>"{, show_unitname};
 BUILDIN(unittalk) {
```

Comment:
bracket for function on new line (i know this not edited by you, but please just include it)